src/rust/ide/view/graph-editor/src/lib.rs

```diff
@@ -1999,7 +2062,60 @@ fn new_graph_editor(app:&Application) -> GraphEditor {
     out.source.edge_target_set <+ new_edge_target;
 
 
+    // == Multi Connect
+    trace out.edge_creation_mode;
+
+    port_input_mouse_up_multi  <- inputs.hover_node_input.sample(&inputs.edge_multi_connect_up).unwrap();
```

Comment:
change the naming a little bit. First part - what is it, second, when it happens, like `port_input_on_up_multi`. I dropped `mouse` and its not guaranteed that it will be mouse event connected here (maybe joystick in VR?)